spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/AbstractServiceRegistrationBean.java

```diff
@@ -56,6 +59,20 @@
     private List<Function<Service<HttpRequest, HttpResponse>,
             ? extends Service<HttpRequest, HttpResponse>>> decorators = new ArrayList<>();
 
+    /**
+     * Sample requests to populate debug forms in {@link DocService}.
```

Comment:
nit: `Sample requests` -> `Example requests`?
We use `sample` and `example` in here. How about migrating `sample` to `example` for keeping consistency. 
WDYT? /cc @trustin @minwoox

Reply:
That's a good idea. Let's use `example`!